shippable.yml

```diff
@@ -8,138 +8,11 @@ matrix:
   exclude:
     - env: T=none
   include:
-    - env: T=sanity/1
```

Comment:
I wouldn't bother with updating `shippable.yml` since it's going away soon: https://github.com/ansible/ansible/pull/72846